### ISSUE TITLE
Error encoding numpy scalars in Python 2.7

### ISSUE THREAD

**Issue by prabhuramachandran** [jsonpickle/jsonpickle#239]
On Python 2.7.13 with jsonpickle 1.0 and numpy 1.13.3, I run into an error with the following:
```
import jsonpickle
import numpy as np
x = np.linspace(0, 1)
jsonpickle.loads(jsonpickle.dumps(x[0]))
```
This fails with a `TypeError` (am happy to upload a full traceback).  It works fine on Python 3.6 and numpy 1.14.1.

**Comment by AndersonReyes** [comment]
@prabhuramachandran you have to register the numpy extension

```
import jsonpickle.ext.numpy as jpnp
jpnp.register_handlers()
```

**Comment by prabhuramachandran** [comment]
Yes that seems to have fixed it.  I am curious why that was not needed on Python 3.6?  That is what tripped me up and made me think this was a bug.

**Comment by AndersonReyes** [comment]
that is a good question hmmmm, seems to be happening at that stage1.__setstate__(state)
